monitoring/grafana/jsonnet/panels/compactor.libsonnet

```diff
@@ -0,0 +1,122 @@
+local grafana = import 'grafonnet/grafana.libsonnet';
+local row = grafana.row;
+local timeseriesPanel = grafana.timeseriesPanel;
+local prometheus = grafana.prometheus;
+
+local span = import 'mixon/span.libsonnet';
+local strings = import 'lib/strings.libsonnet';
+
+local panels = {
+  compaction_success_failure_counts_overview::
+    local expr = |||
+      sum(storage_level_compact_frequency{namespace="$namespace", risingwave_name="$instance"}) by (result)
+    |||;
+
+    timeseriesPanel.new(
+      title='Compaction Success & Failure Counts (Overview)',
+    ).addTarget(
+      prometheus.target(
+        expr=strings.trim(expr),
+        legendFormat='{{ risingwave_component }}',
+      )
+    ) + span.span('half'),
+
+  compaction_success_failure_counts_detailed::
+    local expr = |||
+      sum(storage_level_compact_frequency{namespace="$namespace", risingwave_name="$instance"}) by (pod, group, result)
+    |||;
+
+    timeseriesPanel.new(
+      title='Compaction Success & Failure Counts (Detailed)',
+    ).addTarget(
+      prometheus.target(
+        expr=strings.trim(expr),
+        legendFormat='{{ risingwave_component }}',
+      )
+    ) + span.span('half'),
+
+  max_compaction_task_duration::
+    local expr = |||
+      histogram_quantile(1, sum(rate(state_store_compact_task_duration_bucket{namespace="$namespace", risingwave_name="$instance"}[$__rate_interval])) by (le, pod))
+    |||;
+
+    timeseriesPanel.new(
+      title='Longest Compaction Duration',
+      unit='s',
+    ).addTarget(
+      prometheus.target(
+        expr=strings.trim(expr),
+        legendFormat='{{ risingwave_component }}',
+      )
+    ) + span.span('half'),
+
+  median_compaction_task_duration::
+    local expr = |||
+      histogram_quantile(0.5, sum(rate(state_store_compact_task_duration_bucket{namespace="$namespace", risingwave_name="$instance"}[$__rate_interval])) by (le, pod))
```

Comment:
I think it's ok to use `sum by (le)` instead of `sum by (le, pod)`. I just figured out why we should write like this to get an overall histogram. In short, by `sum by (le, pod)` we will not able to get a result when there're more than 1 compactors.

Reply:
In the original risingwave dashboard, the query looks like this `sum(rate(state_store_compact_task_duration_bucket[1m])) by (le, job, instance))`
What do you think should be the query for our case?